content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -24,24 +24,28 @@ operation:
   > repository, only [DVC-files](/doc/user-guide/dvc-file-format) that are
   > needed to download or reproduce them.
 
-- `.dvc/state`: This file is used for optimization. It is a SQLite database,
+- `.dvc/tmp`: Directory for miscellaneous temporary files
+
+- `.dvc/tmp/index`: Directory for remote index files that are used for
+  optimizing `dvc push/pull/fetch/status -c` operations.
```

Comment:
it creates a link automatically to the commands, better split this into more verbose but clean:

`dvc pull -c`, `dvc push -c` etc